app/controllers/application_controller.rb

```diff
@@ -178,6 +179,14 @@ def initialize_stripe
     Stripe.log_level = Stripe::LEVEL_INFO
   end
 
+  def determine_locale
+    I18n.locale = if %w[en fr].include?(params[:locale])
```

Comment:
> config.i18n.fallbacks = [Settings::UserExperience.default_locale, I18n.default_locale]

I could examine this in more detail to confirm, but I _believe_ that we can't use `Settings::UserExperience` in `application.rb` because that is set on initialization and database-backed information won't yet be available.

That just being an implementation detail, there would be ways around all of this in general. I'll make sure to circle back on the right overall defaults/config as we look to go into prod with this.